Ginger/Ginger/RunSetPageLib/NewRunSetPage.xaml.cs

```diff
@@ -1918,82 +1919,97 @@ private async void xRunRunsetBtn_Click(object sender, RoutedEventArgs e)
             {
                 UpdateRunButtonIcon();
 
-                //UpdateReRunFailedButtonIcon();
+                UpdateReRunFailedButtonIcon();
             }
         }
 
-        //private async void xReRunFailedRunsetBtn_Click(object sender, RoutedEventArgs e)
-        //{
-        //    RunsetExecutor ReRunRunsetExecutor = new RunsetExecutor();
+        private async void xReRunFailedRunsetBtn_Click(object sender, RoutedEventArgs e)
+        {
+            RunsetExecutor ReRunRunsetExecutor = new RunsetExecutor();
+            List<BusinessFlow> DeactivatedBfList = new List<BusinessFlow>();
+            if (WorkSpace.Instance.RunsetExecutor.Runners.Any(x => x.Executor.BusinessFlows.Any(y => y.RunStatus == eRunStatus.Failed)))
```

Comment:
Add else part to indicate there are no failed flows found to rerun

Reply:
Done